kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/filtered-resources/kjars-sources/cloudbalance/pom.xml

```diff
@@ -44,6 +44,13 @@
         <groupId>org.kie</groupId>
         <artifactId>kie-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
It's better to not do it like this, but like this instead:

```
  <properties>
    <maven.compiler.target>1.8</maven.compiler.target>
    <maven.compiler.source>1.8</maven.compiler.source>
  </properties>
```

See org.jboss:jboss-parent pom